### ISSUE TITLE
3.18 support

### ISSUE THREAD

**Issue by ainola** [Neroth/gnome-shell-extension-weather#225]
3.18 is in RC state and will release soon.

Thanks for the extension!

**Comment by ferdnyc** [comment]
Fedora 23 Beta, released two days ago, includes GNOME Shell 3.18. I'm hoping to install it on my "spare" machine soon (this weekend, probably), to determine if the extension is 3.18-compatible. I assume it is, and will just need the `metadata.json` updated to include 3.18 in the list of supported versions.

**Comment by ferdnyc** [comment]
I booted the Fedora 23 Beta live image, which is actually GNOME 3.17.91, and installed the weather extension. As expected, after adjusting the `metadata.json`, the extension worked fine.

Enabling it with gnome-shell-extension-tool produced the following message:
````
% gnome-shell-extension-tool -e weather-extension@xeked.com
/usr/lib64/python3.4/site-packages/gi/overrides/__init__.py:326: Warning: The property 
GSettings:schema is deprecated and shouldn't be used anymore. It will be removed 
in a future version.
  return super_init_func(self, **new_kwargs)
'weather-extension@xeked.com' is now enabled.
````
It's not clear to me whether that was from the extension or the tool, but regardless the extension still enabled successfully and functioned normally.

After testing under gnome-shell 3.17.91, I applied a package update to gnome-shell 3.18.0 and logged in to gnome-shell again. Under 3.18.0 as well, the extension functioned normally with the proper adjustments to `"shell-version"` in `metadata.json`.
